torch/_C/__init__.pyi.in

```diff
@@ -456,8 +456,21 @@ class _InsertPoint:
 class Value:
     def type(self)-> JitType: ...
     def setType(self, t: JitType) -> Value: ...
+    def setTypeAs(self, other: Value) -> Value: ...
+    def inferTypeFrom(self, t: Tensor) -> None: ...
     def debugName(self) -> str: ...
+    def setDebugName(self, name: str) -> None: ...
+    def unique(self) -> _int: ...
+    def offset(self) -> _int: ...
+    def node(self) -> Node: ...
+    def uses(self) -> Iterator[Node]: ...
+    def replaceAllUsesWith(self, val: Value) -> None: ...
+    def replaceAllUsesAfterNodeWith(self, node: Node, val: Value) -> None: ...
     def requires_grad(self) -> _bool: ...
+    def requiresGrad(self) -> _bool: ...
+    def copyMetadata(self, from: Value) -> Value: ...
```

Comment:
Thanks this was because `from` is a keyword and it can't be used as an argument name.
I fixed a similar issue with `is` as a function name later in this same file. Check the latest update.